From: Robert Ancell <robert.ancell@canonical.com>
Date: Mon, 31 Oct 2022 12:08:41 +0100
Subject: portal: Fix broken header guard

This wouldn't have caused an issue with the current header contents, but could have triggered a future bug.

Origin: upstream, 2.74.2, commit:1304f9ed92e6cc2509639de1083e073eec6ab90d
---
 gio/gportalsupport.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gio/gportalsupport.h b/gio/gportalsupport.h
index f7bb6ae..5fe5d82 100644
--- a/gio/gportalsupport.h
+++ b/gio/gportalsupport.h
@@ -19,6 +19,7 @@
  */
 
 #ifndef __G_PORTAL_SUPPORT_H__
+#define __G_PORTAL_SUPPORT_H__
 
 #include <glib.h>
 
